src/components/SingleBlogCard/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Row, Col, Image } from "antd";
+import { NavLink } from "react-router-dom";
+import "./index.css";
+import { useTranslation } from "react-i18next";
+
+const SingleBlogCard = ({ post, order }) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <Row className="justify-content-md-center articleContainer">
+        <div className="containerBlog">
+          <Col lg={{ span: 3, order: order }} className="articleImage">
+            <Image
+              src={post.jetpack_featured_media_url}
+              className="roundImage"
+              roundedCircle
+            />
+          </Col>
+          <Col lg="6" className="blogCardBox">
+            <h3
```

Comment:
Can you change the color of these headers to black as well?

Reply:
@cuygur @ihab-sensei @bertacsevercan shall i change it as black?

Reply:
I agree with changing the headers to black as well.